### DISCUSSION TITLE
Update on the development branch

### DISCUSSION THREAD

**Discussion by kaiyux** [NVIDIA/TensorRT-LLM#2780] [Announcements]
Hi,

The TensorRT-LLM team is pleased to announce that we have pushed an update to the [development](https://github.com/NVIDIA/TensorRT-LLM/tree/main) branch (and the [Triton backend](https://github.com/triton-inference-server/tensorrtllm_backend/tree/main)) this Feb 12, 2025.

This [update](https://github.com/NVIDIA/TensorRT-LLM/pull/2783) includes:

* Model Support
  - Added DeepSeek V3/R1 support on PyTorch workflow.
* Features
  - LLM API
    - Added batched logits processor support.
    - Added EAGLE support.
  - Added FP8 quantization support for Qwen2-VL.
  - Added batched inference support for the LLM API MMLU example `examples/mmlu_llmapi.py`.
  - Added FP4 quantization-layernorm fusion plugin support. (Llama models only)
* API
  - Exposed `kc_cache_retention_config` from C++ `executor` API to the LLM API.

Thanks,
The TensorRT-LLM Engineering Team

**Comment by akhoroshev** [comment]
When will deepseek r1 be available in the trtllm api (not PyTorch)?
